omod/src/main/web/dashboardwidgets/latestobsforconceptlist/latestobsforconceptlist.controller.js

```diff
@@ -6,33 +6,19 @@ export default class LatestObsForConceptListController {
     }
 
     $onInit() {
-        this.maxConceptCount = 10;
         this.maxAgeInDays = undefined;
         this.obs = [];
-
         this.openmrsRest.setBaseAppPath("/coreapps");
-
         this.maxAgeInDays = this.widgetsCommons.maxAgeToDays(this.config.maxAge);
 
-        // Remove whitespaces
-        let concept_list = this.config.concepts.replace(/\s/g,'');
```

Comment:
Can we confirm we don't need this?  ie confirm that the latestobs widget properly strips out whitespace?

Reply:
The code as written actually doesn't strip whitespace. You can see that the `concept_list` variable just gets replaced by something else entirely on line 29. No one seems to have noticed the change in behavior, and I can't actually think of a reason why you would want to strip whitespace from the items in the concept list. Maybe the intention is to `trim`, so that it tolerates whitespace around the comma? Yeah actually I'll add that in; that would make sense to me.

Reply:
yes I was assuming it was to make sure if you had a list like "123, abc, def" it didn't parse it as "123", " abc", " def" and then not find concepts " abc" and " def"